src/mender-update/daemon/state_machine/state_machine.cpp

```diff
@@ -139,6 +139,7 @@ StateMachine::StateMachine(Context &ctx, events::EventLoop &event_loop) :
 
 	// Cannot fail due to FailureMode::Ignore.
```

Comment:
We need to update these comments. The important bit is that only on specific abort error the deployment stops, otherwise is ignored.